filters/auth/grantlogout_test.go

```diff
@@ -0,0 +1,216 @@
+package auth_test
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters/auth"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type testRevokeError struct {
+	Error            string `json:"error"`
+	ErrorDescription string `json:"error_description"`
+}
+
+func newGrantLogoutTestServer() *httptest.Server {
+	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/revoke" {
+			w.WriteHeader(http.StatusNotFound)
+			return
+		}
+
+		expectedCredentials := base64.StdEncoding.EncodeToString([]byte(testClientID + ":" + testClientSecret))
+		expectedAuthorization := "Basic " + expectedCredentials
+		if expectedAuthorization != r.Header.Get("Authorization") {
+			w.WriteHeader(http.StatusUnauthorized)
+			return
+		}
+
+		paramValue := r.URL.Query().Get(testQueryParamKey)
+		if testQueryParamValue != paramValue {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		err := r.ParseForm()
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		token := r.Form.Get("token")
+		tokenType := r.Form.Get("token_type_hint")
+
+		if token == "" || tokenType == "" {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		if tokenType == "access_token" && token == testToken {
+			// Simulate a provider that only supports revoking refresh tokens
+			w.WriteHeader(http.StatusBadRequest)
+			w.Header().Set("Content-Type", "application/json")
+
+			errorResponse := testRevokeError{
+				Error:            "unsupported_token_type",
+				ErrorDescription: "simulate unsupported access token revocation",
+			}
+
+			b, _ := json.Marshal(errorResponse)
+			w.Write(b)
+		} else if tokenType == "refresh_token" && token == testRefreshToken {
+			w.WriteHeader(http.StatusOK)
+		} else {
+			w.WriteHeader(http.StatusUnauthorized)
+		}
+	}))
+}
+
+func checkDeletedCookie(t *testing.T, rsp *http.Response, cookieName string) {
+	for _, c := range rsp.Cookies() {
+		if c.Name == cookieName {
+			if c.Value != "" {
+				t.Fatalf(
+					"Unexpected cookie value, got: '%s', expected: ''.",
+					c.Value,
+				)
+			}
+			if c.MaxAge != -1 {
+				t.Fatalf(
+					"Unexpected cookie MaxAge, got: %d, expected: -1.",
+					c.MaxAge,
+				)
+			}
+			return
+		}
+	}
+
+	t.Fatalf("Cookie not found in response: '%s'", cookieName)
+}
+
+func TestGrantLogout(t *testing.T) {
+	t.Log("create a test provider")
```

Comment:
can you cleanup the logs please?
It's nice to have it while you start working on tests but for the project not. :)

Reply:
Removed the logs here and also in `grant_test.go` and `granclaimsquery_test.go`. Those two were added in the previous PR.